### PR TITLE
Consistent DB upgrade/downgrade arguments

### PR DESCRIPTION
**Pull request by blag**
This is a follow up to #22102, and be forewarned, this might be a bikeshed. If this gets contentious at all, I'll just close it and move on.

I think it's a little bit easier for users to have consistent flags/arguments for the `airflow db upgrade` and `airflow db downgrade` commands. This PR just tweaks the argument processing to expect `--to-revision` and `--to-version` instead of `--revision` and `--version`, respectively.

That change makes the arguments to those commands more consistent with the `--from-revision` and `--from-version` arguments. Doing so also avoids overloading the `--version` flag, which is usually a flag that prints out the version information of the command itself (eg: Airflow's version, which is available via `airflow version`).

An argument against this change is that the `--to-...` arguments can be understood to be implied, like this:

```bash
airflow db upgrade --from-version 10.15.8  # Upgrade from 10.15.8 to the current Airflow version
```

and this means that you do not necessarily need to always specify the `--to-...` arguments. By having both `--to-` and `--from-` arguments, users might think that they always need to specify both a `--to-` and `--from-` argument.

I also fixed an unrelated grammar typo, which corrects the grammar used to log the operation.

### CODE REVIEW

File: airflow/cli/commands/db_command.py [hunk 1]
```diff
@@ -49,15 +49,15 @@ def resetdb(args):
...
     print("DB: " + repr(settings.engine.url))
-    if args.revision and args.version:
-        raise SystemExit("Cannot supply both `--revision` and `--version`.")
+    if args.to_revision and args.to_version:
+        raise SystemExit("Cannot supply both `--to-revision` and `--to-version`.")
     if args.from_version and args.from_revision:
         raise SystemExit("Cannot supply both `--from-revision` and `--from-version`")
     if (args.from_revision or args.from_version) and not args.show_sql_only:
         raise SystemExit(
             "Args `--from-revision` and `--from-version` may only be used with `--show-sql-only`"
```
Comment (ephraimbuddy) [inline airflow/cli/commands/db_command.py:58]: Looks like when we use `--from-version`  or `--from-revision` we must use `--show-sql-only`. Should we make it to run real migration if the flag is used and raise error only if the user's version/revision is not the same with the supplied `--from-revision` /`--from-version`

Comment (blag) [reply airflow/cli/commands/db_command.py:58]: That's a good idea, but it's outside the scope of this PR.

Comment (dstandish) [reply airflow/cli/commands/db_command.py:58]: my 2c, that would make it so that, when used without `sql-only`, the arg `--from-version` would only be allowed when it has _no effect_; in that case it seems to me it makes sense to just not accept it.    is there some benefit to accepting the arg in that case?
it would make the documentation more cumbersome too.

**Review by dstandish — APPROVED** [review]
no objections from me

File: airflow/utils/db.py [hunk 2]
```diff
@@ -742,7 +742,7 @@ def check_and_run_migrations():
     if len(db_heads) < 1:
         db_command = initdb
         command_name = "init"
-        verb = "initialization"
+        verb = "initialize"
```
Comment (dstandish) [inline airflow/utils/db.py:745]: why is this changed here? did something change?

Comment (dstandish) [reply airflow/utils/db.py:745]: maybe it was renamed in alembic with deprecation?

Comment (blag) [reply airflow/utils/db.py:745]: That's the unrelated grammar typo I mentioned in the description ("initialization" isn't a verb).

If this PR isn't merged, I'll submit a tiny fix for that separately. It just seemed a little heavy handed to open a full PR for a tiny typo fix.

Comment (dstandish) [reply airflow/utils/db.py:745]: ohhh 💡 

i thought that `verb` was used for more than console output like `getattr(db, verb)()` or something

but no, that's what `command_name` is i guess.  so cool really just a typo 👍

**Review by kaxil — APPROVED** [review]

**Comment by blag** [comment]
This seems a lot less contentious than I was expecting it to possibly be, so I think it's ready to be merged.

**Review by jedcunningham — APPROVED** [review]

**Review by potiuk — APPROVED** [review]
